src/Controls/src/Core/VisualStateManager.cs

```diff
@@ -663,6 +664,22 @@ internal static IList<VisualStateGroup> Clone(this IList<VisualStateGroup> group
 
 			return actual;
 		}
+
+		internal static VisualState FindFirstVisualStateByName(this VisualElement element, string name)
```

Comment:
VisualState names have to be unique, so you can just call this `FindVisualStateByName`. Or better yet, just `bool HasVisualState()`.